scripts/create_perf_json.py

```diff
@@ -1557,32 +1569,24 @@ def append_to_desc(s: str):
                 threshold = threshold.replace('& 1', '')
                 thresholds[i.name] = threshold
                 _verboseprint2(f'{i.name} -> {threshold}')
-            save_form(i.name, i.groups, form, i.desc, i.locate, i.scale_unit,
-                      threshold, i.issues)
-
-        if 'Socket_CLKS' in infoname:
-            form = 'Socket_CLKS / #num_dies / duration_time / 1000000000'
-            form = resolve_all(form, expand_metrics=False)
-            if form:
-                formula = metric.ParsePerfJson(form)
-                save_form('UNCORE_FREQ', 'SoC', formula.ToPerfJson(),
-                          'Uncore frequency per die [GHZ]', None, None, None, [])
+            self.save_form(i.name, i.groups, form, i.desc, i.locate, i.scale_unit,
+                           threshold, i.issues, pmu_prefix, events, infoname, aux,
+                           issue_to_metrics, saved_formulas)
 
+    def extract_extra_metrics(self, pmu_prefix: str, events: Dict[str, PerfmonJsonEvent],
+                              saved_formulas: list[Dict[str, str]]):
         if 'extra metrics' in self.files:
             with open(self.files['extra metrics'], 'r') as extra_json:
+                ignore = {
+                    'tma_info_system_mux': 'MUX',
+                    'tma_info_system_power': 'Power',
+                    'tma_info_system_time': 'Time',
+                }
                 broken_metrics = {
-                    'ICX': {
-                        # Missing event: UNC_CHA_TOR_INSERTS.IA_MISS_LLCPREFDRD
-                        'llc_data_read_mpi_demand_plus_prefetch',
-                        # Missing event: UNC_CHA_TOR_OCCUPANCY.IA_MISS_DRD_DRAM
-                        'llc_demand_data_read_miss_to_dram_latency',
-                        # Missing event: UNC_CHA_TOR_INSERTS.IO_HIT_RDCUR
-                        'io_bandwidth_read',
+                    'EMR': {
+                        # Missing event: UNC_UPI_TXL_FLITS.ALL_DATA
```

Comment:
These event names are a bit unfortunate since they diverge from the expected uppercase, numbers, underscores, and periods. Does SPR also bump into this mismatch in `TXL vs. TxL`?